src/components/Resource/Wizard/ModelWizard.tsx

```diff
@@ -270,13 +280,17 @@ function getWizardUrl({
   id,
   step,
   templateId,
+  bountyId,
 }: {
   step: number;
   id?: number;
   templateId?: number;
+  bountyId?: number;
 }) {
   if (!id) return '';
-  return `/models/${id}/wizard?step=${step}${templateId ? `&templateId=${templateId}` : ''}`;
+  return `/models/${id}/wizard?step=${step}${templateId ? `&templateId=${templateId}` : ''}${
+    bountyId ? `&bountyId=${bountyId}` : ''
+  }`;
```

Comment:
Since we are adding more values to the query string, we can benefit from using `QS.stringify` instead
```suggestion
const queryString = QS.stringify({ templateId, bountyId, step }); 
return `/models/${id}/wizard?${queryString}`;
```